python/pyspark/sql/functions.py

```diff
@@ -299,6 +302,22 @@ def countDistinct(col, *cols):
     return Column(jc)
 
 
+@since(1.5)
+def e():
```

Comment:
actually even in java/scala it is probably not as useful. i think we should just remove those from the dataframe api, but leave the expression there for sql.